### ISSUE TITLE
New Challenge Mode times overwrite Reverse times

### ISSUE THREAD

**Issue by blindseerjb** [hasore/DragonRidingRace#8]
I know the Challenge and Reverse Challenge modes are new, but I noticed today when I completed Morqut Ascent on the Challenge mode, the addon announced a new time for the Reverse version of the Advanced track.

Please let me know if I can provide any additional details. I really love this addon and want to help if I can. Thank you!!

**Comment by hasore** [comment]
Was the saved time updated?

**Comment by blindseerjb** [comment]
Yes, the Challenge mode time was saved as the Reverse time. My Reverse time was originally 51.336, but now shows as the Challenge time of 55.521.
![Uploading Morqut Ascent.png…]()

**Comment by blindseerjb** [comment]
When rerunning the Reverse course, the Reverse entry reverts to my best Reverse time.

**Comment by hasore** [comment]
I will take a look over it this weekend.

**Comment by blindseerjb** [comment]
Thank you!! Let me know if I can provide any additional data.

**Comment by blindseerjb** [comment]
Popping in to check up on this issue, and make sure everything with you is good. No rush for an update, just wanted to make sure you were doing well.
